### PR TITLE
Simplify description language

### PR DESCRIPTION
**Pull request by brianteeman**
It is important that the text we use in the UI is understandable. As many people will be using Joomla in English even if it is not their most proficient language simpler language is more useful. It is also part of the new WCAG 2.1 accessibility guidelines for cognitive ability.

An online reading level analyser rated the current text as a reading level of grade 9 and the new text as reading level of grade 4

This reading level measurement actually gauges the lowest education needed to understand your prose using the The Automated Readability Index

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_system_updatenotification.ini [hunk 1]
```diff
@@ -17,7 +17,7 @@ PLG_SYSTEM_UPDATENOTIFICATION_EMAIL_DESC="A comma separated list of the email ad
...
 PLG_SYSTEM_UPDATENOTIFICATION_LANGUAGE_OVERRIDE_LBL="Email Language"
-PLG_SYSTEM_UPDATENOTIFICATION_LANGUAGE_OVERRIDE_DESC="If you choose Auto (default), the update notification email to Super Users will be in the site language at the time the plugin is triggered. By selecting a language here you are forcing the update notification emails to be sent in this specific language."
+PLG_SYSTEM_UPDATENOTIFICATION_LANGUAGE_OVERRIDE_DESC="Select a language for the update notification emails or set to Auto to send them in the site language at the time."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_system_updatenotification.ini]: How about `emails. Set to` to break up the sentence and make the sentences short?

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.plg_system_updatenotification.ini]: I think it is short enough but I am happy to make the change

File: administrator/language/en-GB/en-GB.plg_system_actionlogs.ini [hunk 2]
```diff
@@ -11,7 +11,7 @@ PLG_SYSTEM_ACTIONLOGS_INFO_LABEL="Information"
 PLG_SYSTEM_ACTIONLOGS_JOOMLA_ACTIONLOG_DISABLED="Action Log - Joomla"
 PLG_SYSTEM_ACTIONLOGS_JOOMLA_ACTIONLOG_DISABLED_REDIRECT="The %s plugin is disabled."
 PLG_SYSTEM_ACTIONLOGS_LOG_DELETE_PERIOD="Days to delete logs after"
-PLG_SYSTEM_ACTIONLOGS_LOG_DELETE_PERIOD_DESC="Enter how many days logs should be kept before they are deleted. Enter 0 if you don't want to delete the logs."
+PLG_SYSTEM_ACTIONLOGS_LOG_DELETE_PERIOD_DESC="Enter 0 if you don't want to delete the logs."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_system_actionlogs.ini:14]: Maybe `Enter 0 to not delete the logs.`?

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.plg_system_actionlogs.ini:14]: No I am happy with it the way it is.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on eb420adb6e88a3298248709cae6f8dec8ad5351f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25844">issues.joomla.org/tracker/joomla-cms/25844</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on eb420adb6e88a3298248709cae6f8dec8ad5351f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25844">issues.joomla.org/tracker/joomla-cms/25844</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by zero-24** [comment]
Thanks @brianteeman reads more clear now 👍

**Comment by brianteeman** [comment]
thx

**Pull request merged**